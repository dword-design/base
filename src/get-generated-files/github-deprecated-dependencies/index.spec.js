import self from '.'

export default {
  valid() {
    expect(self).toMatchSnapshot(this)
  },
}
