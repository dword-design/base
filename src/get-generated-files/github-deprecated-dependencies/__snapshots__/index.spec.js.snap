// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index valid 1`] = `
Object {
  "jobs": Object {
    "run": Object {
      "runs-on": "ubuntu-latest",
      "steps": Array [
        Object {
          "uses": "actions/checkout@v4",
          "with": Object {
            "lfs": true,
          },
        },
        Object {
          "continue-on-error": true,
          "id": "check-deprecated-js-deps",
          "uses": "tinovyatkin/action-check-deprecated-js-deps@v1",
        },
        Object {
          "env": Object {
            "DEPRECATED": "\${{ steps.check-deprecated-js-deps.outputs.deprecated }}",
            "GITHUB_TOKEN": "\${{ secrets.GITHUB_TOKEN }}",
            "RUN_URL": "https://github.com/\${{github.repository}}/actions/runs/\${{github.run_id}}",
          },
          "id": "create-deprecation-issue",
          "if": "\${{ steps.check-deprecated-js-deps.outputs.deprecated }}",
          "uses": "JasonEtco/create-an-issue@v2",
          "with": Object {
            "filename": ".github/DEPRECATED_DEPENDENCIES_ISSUE_TEMPLATE.md",
            "update_existing": true,
          },
        },
        Object {
          "if": "\${{ !steps.check-deprecated-js-deps.outputs.deprecated && steps.create-deprecation-issue.outputs.number }}",
          "uses": "peter-evans/close-issue@v3",
          "with": Object {
            "comment": "Auto-closing the issue",
            "issue-number": "\${{ steps.create-deprecation-issue.outputs.number }}",
          },
        },
        Object {
          "uses": "gautamkrishnar/keepalive-workflow@v1",
        },
      ],
    },
  },
  "name": "deprecated-dependencies",
  "on": Object {
    "schedule": Array [
      Object {
        "cron": "0 5 * * MON",
      },
    ],
  },
}
`;
